Minor changes after code review

jweisman opened the pull request:
1. Add class to buttons in configuration
2. Add scope to URL in configuration
3. Hide links if no entity selected
4. Remove hard-coded scope from URL
5. Filter and handle items or bib records
6. Handle any NZ number
7. Add subtitle, entities, and popups to manifest

jweisman commented on cloudapp/src/app/main/main.component.ts:
```diff
@@ -20,7 +20,7 @@ export interface Criteria {
   
   private pageLoad$: Subscription;
-  base_url: string;
+  base_url: string = "";
```
This was causing an error when the app is loaded before the base_url is initialized

jweisman commented:
Nice app! Thanks for your submission. This PR has a few suggestions for your consideration.

We think this can be useful to others as well, so we're suggesting ways to make it more generic. Happy to hear what you think.

Thanks.

FNGM42 commented:
Josh,
Really happy you like the app.  Just want to make sure I have things straight regarding these suggestions.

1. Add class to buttons in configuration <- this looks already done, yes? or is this for each button as opposed to do div?
2. Add scope to URL in configuration <- self explanatory; make sure I have the NZ scope as a configuration part for other consortia
3. Hide links if no entity selected <- do you mean if the app is open on say, the Alma home page?
4. Remove hard-coded scope from URL <- part of #2
5. Filter and handle items or bib records <-you mean check for MMSID and/or Barcode?
6. Handle any NZ number <- need a bit of clarification on this
7. Add subtitle, entities, and popups to manifest <-same here

jweisman commented:
Hi @FNGM42 - All of the changes suggested were done in this pull request. So now that you've merged the request the app should be ready to be published. (I believe you're communicating with @mgobat on that). 

Below I can point out the lines where each change was made so you can see what was done:

1. [Here](https://github.com/GIL-GALILEO/last-copy-widget/pull/1/files#diff-1acbcb0357df588538875ecc5c6048c97cfe6b0d53ac2c8e27639ba1f1f2e2d4R1)
2. [Here](https://github.com/GIL-GALILEO/last-copy-widget/pull/1/files#diff-bbe8a36e035d55f4cf2a2acb02f2d417b0be5da0ea125d8d994c89eb58800d5bR78)
3. Right- so if there are no BIBs or items displayed, the app won't display a link now. [Here](https://github.com/GIL-GALILEO/last-copy-widget/pull/1/files#diff-969cd7162e9be13f1e6be3bf3cf8eb3400786fe6b987023aaba353f9aa119db2R20)
4. Correct- part of #2
5. Right- it could be that your app is open on another page, like PO lines or users. So this is a best practice to make sure you're interacting only with entities you know how to deal with. [Here](https://github.com/GIL-GALILEO/last-copy-widget/pull/1/files#diff-bbe8a36e035d55f4cf2a2acb02f2d417b0be5da0ea125d8d994c89eb58800d5bR141)
6. This change looks for any network number with a EXLNZ- prefix rather than only EXLNA-01GALI..... Just makes it more generic. [Here](https://github.com/GIL-GALILEO/last-copy-widget/pull/1/files#diff-bbe8a36e035d55f4cf2a2acb02f2d417b0be5da0ea125d8d994c89eb58800d5bR184)
7. Made a few changes to the manifest ([here](https://github.com/GIL-GALILEO/last-copy-widget/pull/1/files#diff-ffa5b716b5a57837f7929dfcca4b4dfdeb97210a7fd5a12d2f1978846d6f1743R19)). More information on the manifest fields is available [here](https://developers.exlibrisgroup.com/cloudapps/docs/manifest/).
  * Added a subtitle field which is shown in the app list
  * Added a security declaration which allows popups to be opened (like the Primo link)
  * Added a list of entities that the app supports (BIBs and Items) so that Alma can suggest the app when a user is on a page which shows those entities.

Hope these explanations make it more clear. Again- thanks for your submission. Looking forward to seeing the app in the App Center.
